### ISSUE TITLE
Quickblox with AFNetworking on the same project

### ISSUE THREAD

**Issue by nelsonrgpinto** [QuickBlox/quickblox-ios-sdk#546]
Hi,

I'm trying to install, for the first time, the latest Quickblox iOS SDK via Cocoapods on a project that uses AFNetworking. When running the app, it crashes in AFNetworking's AFURLSessionManager (af_resume method). If i removed the pod everything goes back to normal.
Any help on this would be great. Thanks.

XCode 7.3.1 - Latest version o Cocoapods installed. Testing on iOS 9.3.

https://gist.github.com/nelsonrgpinto/84693a9338eb73248eec4f09958ecec6

**Comment by Raikerian** [comment]
Hi @nelsonrgpinto 

It is indeed a strange crash, can you provide some code snippets in order for us to reproduce it? Maybe some test project?

**Comment by nelsonrgpinto** [comment]
Hi @Raikerian 

I'm not able to provide some code snippets because i didn't even started coding. I just integrated Quickblox using `pod install` and ran the App. But i tested on a simple project, installing Quickblox and AFNetworking via Cocoapods and the app didn't crash. Then i decided to integrate AFNetworking via Cocoapods on my real project and now everything works fine. I also migrated the code to the latest version of AFNetworking.
Thanks.
